Functions.Templates/Templates/EventGridBlobTrigger-CSharp-Isolated/.template.config/template.json

```diff
@@ -0,0 +1,77 @@
+{
```

Comment:
Can you rename the directory to `EventGridBlobTrigger-CSharp-Isolated-6.x`

Reply:
Renamed